cloudify_marketplace/README.md

```diff
@@ -3,7 +3,7 @@
 ## Prerequisites
 > A UNIX-like OS is required for running Packer. Builds aren't supported on Windows
 
-* [packer](https://www.packer.io/intro/getting-started/setup.html) (tested on 0.8.6)
+* [packer](https://www.packer.io/intro/getting-started/install.html) (tested on 1.2.5)
```

Comment:
Build vm's use packer 0.10.0

Reply:
What is the reason don't use latest? 1.2.5?